nodejs/week1/warmup.js

```diff
@@ -0,0 +1,32 @@
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+
+    getDiameter() {
```

Comment:
typically a get function returns something. Here you're setting a property instead. I'd recommend NOT using .diameter but instead just `return 2*this.radius`